retrieval/target.go

```diff
@@ -88,11 +89,8 @@ func (t TargetHealth) value() model.SampleValue {
 }
 
 const (
-	// HealthUnknown is the state of a Target before it is first scraped.
```

Comment:
Probably – single line above `const` sounds good though. Just not that excited by doc strings consisting of spaced combinations of identifier names.